JavaAPI/pom.xml

```diff
@@ -136,6 +136,11 @@
             <artifactId>log4j-core</artifactId>
             <version>2.14.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
```

Comment:
We do not need slf4j, it is for logging: we are using log4j instead so make sure you have that dependency

Reply:
So, remove this dependency

Reply:
log4j needs that dependency Marc